kie-server-parent/kie-server-controller/kie-server-controller-websocket/src/main/java/org/kie/server/controller/websocket/WebSocketKieServerControllerImpl.java

```diff
@@ -76,14 +77,14 @@ public WebSocketKieServerControllerImpl() {
     
     @PostConstruct
     public void configure() {
-        try {
-            setTemplateStorage(templateStorage.get());
-        } catch (RuntimeException e) {
+        if(templateStorage.isUnsatisfied()){
             logger.warn("Unable to find template storage implementation, using in memory");
+        } else {
+            setTemplateStorage(templateStorage.get());
         }
-        try {
+        if(notificationService.isUnsatisfied()) {
             setNotificationService(notificationService.get());
```

Comment:
Shouldn't be here a warning log and in else branch actual code - setNotificationService(notificationService.get());

Reply:
That's right logic in here is inverted, will fix it.